index.html

```diff
@@ -331,6 +331,8 @@
                   painted, and if the <code><a>frameCaptureRequested</a></code>
                   internal property of <var>track</var> is set, add a new frame
                   to <var>track</var> containing what was painted to the canvas.
+                  The captured frame should preserve the alpha channel if canvas
```

Comment:
"if *the* canvas"

This probably isn't sufficiently imperative.  We need to be definite about what does and doesn't happen.  "should" allows too much variation.